dev/_sass/_project-ui.scss

```diff
@@ -66,14 +66,6 @@
   background: transparent !important;
 }
 
-.section {
-  margin-bottom: $section-distance !important;
-}
-
-.img-responsive-inline {
-  @include img-responsive(inline-block);
-}
-
```

Comment:
Why those styles have been removed?

Reply:
This mixin included from bootstrap v3 mixins

Reply:
Doesn't this className usable anymore?
Or you've moved to another place?

Reply:
Bootstrap v4 has new mixin `img-fluid`

Reply:
Got it!